packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -91,18 +91,18 @@ export const TranslationSidebar = forwardRef<
 
     const [translatorNoteContent, setTranslatorNoteContent] = useState('');
     const [footnoteContent, setFootnoteContent] = useState('');
-    const wordId = useRef('');
+
     useEffect(() => {
-      if (notesQuery.isSuccess && word.id !== wordId.current) {
-        wordId.current = word.id;
```

Comment:
We still need these changes. It ensures we only reset the note state when the word id changes

Reply:
I'm a little confused. As I understand, `notesData` (which is just `notesQuery.data`) will only change when:
1. The `language` changes
2. The `word.id` changes
3. New data is fetched

Wouldn't we want to update the notes content in all these cases (or at least 1 and 2)? It seems like the only reason we needed the `wordId` ref was because the `notesQuery` object, which was used as a dependency instead of the `notesData`, changes every render.

Reply:
We are trying to prevent the 3rd case. After autosave, we fetch the latest version of the notes, but that might be a few characters behind the current value of the text input after the network requests have settled. The wordId ref prevents the notes from being reset if the update was due to an autosave rather than a word change. This works for word changes, but not for languages. That's ok for now. Turns out the work on phrases is going to fix this. So for now, let's leave this as it was

Reply:
Done